helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/constraints/SoftConstraintWeightModel.java

```diff
@@ -26,11 +26,15 @@
 /**
  * The class retrieves the offline model that defines the relative importance of soft constraints.
  */
-class SoftConstraintWeightModel {
+public class SoftConstraintWeightModel {
```

Comment:
I think this class shall only be used inside the constraint package. If any other classes need to know it, our OO design has some issues.